handler/setup.go

```diff
@@ -95,6 +96,7 @@ func HandleSetup() http.HandlerFunc {
 
 		// fmt.Printf("setup request config: %+v\n", s.SetupRequestConfig)
 		s.Volumes = append(s.Volumes, getSharedVolume())
+		s.Volumes = appendConfigVolume(s.Volumes)
```

Comment:
let the function return the volumes from the config, and do the appending here. Appending a list in different functions makes it a bit hard to follow